src/client/api_util.cpp

```diff
@@ -1847,9 +1847,10 @@ AttributeInfoEx &AttributeInfoEx::operator=(const AttributeConfig_5 *att_5)
 		else
 			memorized = MEMORIZED_WRITE_INIT;
 	}
+	enum_labels.clear();
 	for (unsigned int j=0; j<att_5->enum_labels.length(); j++)
```

Comment:
Please note that there are many lines of code to change in order to follow this standard naming pattern. I changed this for loop line but I won't change all the other similar lines in this pull request.